atlite/cutout.py

```diff
@@ -126,6 +126,14 @@ def __init__(self, path, **cutoutparams):
         gebco_path: str
             Path to find the gebco netcdf file. Only necessary when including
             the gebco module.
+        esgf_params: dict
+            Parameters to be used in search on the ESGF database.
+        model: str
```

Comment:
If we keep that we should rename this keyword argument into something more specific, like esgf_model